### ISSUE TITLE
bug: analytics not working on android

### ISSUE THREAD

**Issue by MitchellPeck** [capawesome-team/capacitor-firebase#369]
### Plugin(s)

- [X] Analytics
- [ ] App
- [ ] App Check
- [ ] Authentication
- [ ] Crashlytics
- [ ] Cloud Messaging
- [ ] Performance
- [ ] Remote Config

### Did you test the latest version?

- [X] I use the latest version

### Platform(s)

- [X] Android
- [ ] iOS
- [ ] Web

### Current behavior

Analytics sends data when I open the app in a web browser, even on an Android device. But when I open the app on an Android device, no data is sent.

### Expected behavior

Data should be sent from the Android device when I open the app on said Android device.

### Reproduction

https://github.com/MitchellPeck/bug-analytics-not-working-on-android

### Steps to reproduce

1Export the app for android
2Open the app on an android device
3Check that analytics data hasn't been sent

### Capacitor doctor

💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 4.7.3
  @capacitor/core: 4.7.3
  @capacitor/android: 4.7.3
  @capacitor/ios: 4.7.3

Installed Dependencies:

  @capacitor/cli: 4.7.0
  @capacitor/android: 4.7.0
  @capacitor/ios: 4.7.0
  @capacitor/core: 4.7.0

[success] Android looking great! 👌

### Before submitting

- [X] I understand that incomplete issues (e.g. without reproduction) are closed.

**Comment by robingenz** [comment]
Thank you for your request. I will have a look as soon as possible.

**Comment by robingenz** [comment]
I tested it myself and also had difficulties with Android at first. But in the end it was only because the app had transferred the events only after a few hours. The next day I had all events in Google Analytics.
From the [Firebase documentation](https://firebase.google.com/docs/analytics/debugview):

> Generally, events logged by your app are batched together over the period of approximately one hour and uploaded together. This approach conserves the battery on end users’ devices and reduces network data usage.

But for debugging there is the option to activate the [Debug Mode](https://firebase.google.com/docs/analytics/debugview#enable_debug_mode). With this I was able to follow the events live.

![grafik](https://user-images.githubusercontent.com/13857929/235755154-788131a3-8387-4cf5-afdf-1f7154d6fa23.png)
